packages/schemas/src/seeds/sign-in-experience.ts

```diff
@@ -1,14 +1,17 @@
 import { Language } from '@logto/phrases-ui';
+import { generateDarkColor } from '@logto/shared';
 
 import { CreateSignInExperience, SignInMode } from '../db-entries';
 import { BrandingStyle, SignInMethodState } from '../foundations';
 
+const defaultPrimaryColor = '#6139F6';
+
 export const defaultSignInExperience: Readonly<CreateSignInExperience> = {
   id: 'default',
   color: {
-    primaryColor: '#6139F6',
-    isDarkModeEnabled: false,
-    darkPrimaryColor: '#8768F8',
+    primaryColor: defaultPrimaryColor,
+    isDarkModeEnabled: true,
```

Comment:
should be false per our discussion with guamian

Reply:
Yes, is a mistake on resolving conflict